doc/api/net.md

```diff
@@ -226,6 +226,11 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: Depending on the operating system, listening to an IPv6 address such
+as `'::'` may cause the `net.Server` to listen on an equivalent IPv4 address
```

Comment:
It seems only for `::` have this phenomenon.

| OS                  | Listen IPv6 address | Does equivalent IPv4 auto listened
| ------------- | -------------------- | ---------------------------------- |
| Windows 7Pro/10 | :: | YES
| Mac OS X EI Capitan 10.11.6 | :: | YES
| Ubuntu 14.04/16.04 | :: | YES
| Windows 7Pro/10 | ::1 | NO
| Mac OS X EI Capitan 10.11.6 | ::1 | NO
| Ubuntu 14.04/16.04 | ::1 | NO
| Windows 7Pro/10 | ActualIPv6Address | NO
| Mac OS X EI Capitan 10.11.6 | ActualIPv6Address | NO
| Ubuntu 14.04/16.04 | ActualIPv6Address | **Unknown** (failed to bind (invalid parameter)